src/_macosx.m

```diff
@@ -1,3 +1,4 @@
+#include <objc/NSObjCRuntime.h>
```

Comment:
What functions are we grabbing from the runtime header? Everything below looks pretty standard I thought.

Reply:
That's weird, I don't remember adding this... going to revert.